sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/DatabaseAccountOperationsTests.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    [TestFixture]
+    public class DatabaseAccountOperationsTests : CosmosDBManagementClientBase
+    {
+        private string resourceGroupName;
+        private const string databaseAccountName = "db4937";
```

Comment:
I thought we were doing Recording.GenerateAssetName to get a random name for the database to reduce the chances that the name is already taken.

Reply:
Hardcode a generated random name for now, future framework changes will make this obsolete.